api/ask_astro/slack/controllers/mention.py

```diff
@@ -34,116 +95,35 @@ async def on_mention(
         logger.error(f"Missing key: {e}")
         return
 
-    logger.info("Received question (%s): %s", ts, text)
+    logger.info(f"Received question ({ts}): {text}")
```

Comment:
I think we'd better consolidate how we use logger

Reply:
Changed it